workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/DotUtil.java

```diff
@@ -158,14 +162,14 @@ public static StringBuffer getDashboardDot(Collection<OozieNode> oozieNodes, Lis
             if (processInfo.getParentProcessId() == 0) {
                 String[] nextProcess = processInfo.getNextProcessIds().split(",");
                 if (nextProcess.length > 1) {
-                    dot.append("fork_" + processInfo.getNextProcessIds().replace(',', '_') + " [shape=rnastab,label =\" \",style=\"rounded\",penwidth=.25,height = 0.1,width =0.1,color=gray86];\n");
-                    dot.append("start" + "->" + "fork_" + processInfo.getNextProcessIds().replace(',', '_') + " ;\n");
+                    dot.append(FORK + processInfo.getNextProcessIds().replace(',', '_') + " [shape=rnastab,label =\" \",style=\"rounded\",penwidth=.25,height = 0.1,width =0.1,color=gray86];\n");
+                    dot.append("start" + "->" + FORK + processInfo.getNextProcessIds().replace(',', '_') + " ;\n");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "start" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)